cdap-ui/app/features/tracker/tracker.less

```diff
@@ -545,6 +544,197 @@ body.state-tracker-help {
   }
 }
 
+body.state-tracker-integrations {
+  background-color: white;
+
+  .integration-item {
+    margin: 0;
+    padding: 10px 25px;
+    border-bottom: 1px solid @border-color;
+    height: 80px;
+    width: 100%;
+    display: table;
+
+    .integration-information {
+      width: 50%;
```

Comment:
Do we need the styles for ```integration-information``` and ```integration-setup```? Wouldn't ```pull-left``` and ```pull-right``` work? Tried pullings things to left and right and seemed to work ok. I understand font-color and margin but is there any specific reason for the table-cell display?